contrib/README.md

```diff
@@ -0,0 +1,121 @@
+# How to start the lorri daemon as a service
+
+This guide shows you how to run `lorri daemon` as a service. The exact steps
+depend on your operating system and general setup. Currently, we have
+instructions for these setups:
+
+- [Run `lorri daemon` on NixOS](#run-lorri-daemon-on-nixos)
+- [Run `lorri daemon` on Linux with
+  home-manager](#run-lorri-daemon-on-linux-with-home-manager)
+- [Run `lorri daemon` on Linux with just
+  systemd](#run-lorri-daemon-on-linux-with-just-systemd)
+- [Run `lorri daemon` on macOS with
+  home-manager](#run-lorri-daemon-on-macOS-with-home-manager)
+
+## Run `lorri daemon` on NixOS
+
+On NixOS, first copy [`lorri.nix`] into your NixOS configuration
+directory:
```

Comment:
We have the NixOS service in nixpkgs now, so we can get rid of this methinks?

Reply:
I partially agree. This should all become "put `services.lorri.enable = true;` in your `configuration.nix`. Unfortunately right now that only works for those brave enough to run NixOS `master` (does anyone?) - so that will only be widely applicable once `lorri` is in NixOS stable, methinks.

In the meantime, I'm not sure what we should suggest. Perhaps we can point to the package and service definition on nixpkgs and ask people to get them from there? Either way, I agree with you that the package and service nix files don't belong in this PR anymore now that improved versions of them are in nixpkgs.

Reply:
I think it's fine to backport non-breaking changes (additions) to the stable release branch, too.

For example, `iwd` support for `NetworkManager` was backported to 19.09.

Reply:
Okay, I think we only improve the situation if we backport (right now the `lorri` attribute in nixpkgs 19.09 throws an error).

Reply:
https://github.com/NixOS/nixpkgs/pull/73451